enhancements/observability/multicluster-observability-addon.md

```diff
@@ -0,0 +1,712 @@
+---
+title: multicluster-observability-addon
+authors:
+  - "@periklis"
+  - "@JoaoBraveCoding"
+  - "@pavolloffay"
+  - "@iblancasa"
+reviewers:
+  - "@alanconway"
+  - "@jcantrill"
+  - "@xperimental"
+  - "@jparker-rh"
+  - "@berenss"
+  - "@bjoydeep"
+approvers:
+  - "@alanconway"
+  - "@jcantrill"
+  - "@xperimental"
+api-approvers:
+  - "@alanconway"
+  - "@jcantrill"
+  - "@xperimental"
+creation-date: 2023-11-28
+last-updated: 2023-11-28
+tracking-link:
+  - https://issues.redhat.com/browse/OBSDA-356
+  - https://issues.redhat.com/browse/OBSDA-393
+  - https://issues.redhat.com/browse/LOG-4539
+  - https://issues.redhat.com/browse/OBSDA-489
+see-also:
+  - None
+replaces:
+  - None
+superseded-by:
+  - None
+---
+
+# Multi-Cluster Observability Addon
```

Comment:
As per now we intend and plan to deliver the whole addon (collection and storage) in separate stages, we should rename this proposal to what it actually is: `Multi-Cluster Logs and Traces Forwarding`.

Reply:
I would say no for now because metric collection is part of MCO Operator. @moadz and et.al. need to outline the migration path and in turn which CRDs to use to metric collection in MCOA.